xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp

```diff
@@ -1187,8 +1187,15 @@ int pre_header_feeding(am_private_t *para, am_packet_t *pkt)
         } else if ((CODEC_TAG_WVC1 == para->video_codec_tag)
                 || (CODEC_TAG_VC_1 == para->video_codec_tag)
                 || (CODEC_TAG_WMVA == para->video_codec_tag)) {
-            CLog::Log(LOGDEBUG, "CODEC_TAG_WVC1 == para->video_codec_tag");
-            ret = wvc1_write_header(para, pkt);
+            if (para->extrasize > 4 && !*para->extradata && !*(para->extradata + 1) &&
+                *(para->extradata + 2) == 0x01 && *(para->extradata + 3) == 0x0f && ((*(para->extradata + 4) & 0x03) == 0x03)) {
```

Comment:
curly bracket has to go into a new line, see https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-braces